modules/boost.accumulators/1.83.0.bzl.1/MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+module(
+    name = "boost.accumulators",
+    version = "1.83.0.bzl.1",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "boost.rules.tools", version = "1.0.0")
+bazel_dep(name = "platforms", version = "0.0.8")
+bazel_dep(name = "boost.array", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.assert", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.circular_buffer", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.concept_check", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.config", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.core", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.fusion", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.iterator", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.mpl", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.numeric.conversion", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.numeric.ublas", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.parameter", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.preprocessor", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.range", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.serialization", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.static_assert", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.throw_exception", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.tuple", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.type_traits", version = "1.83.0.bzl.1")
+bazel_dep(name = "boost.typeof", version = "1.83.0.bzl.1")
+
+# Test Only
+bazel_dep(name = "boost.test", version = "1.83.0.bzl.1")
```

Comment:
Could these be dev dependencies if they are only used for testing?

Reply:
These test only ones are used by the unit tests for the module. Maybe that would work if the Bazel Central Registry still includes them to run the tests, but then they aren't needed by the end user?

Reply:
Unit tests are in the test folder of the source, the patch patches in a build file that runs them

Reply:
Yes, if you want the BCR presubmit to the run the tests, these need to be non-dev dependencies. Alternatively, you could make the test directories into BCR test modules with their own dependencies, but that would likely be much more effort. 

I don't think the few additional deps matter much in the end, so feel free to leave them in!

Reply:
Awesome! I've only just started adding tests, but looks like there's not too many test-only deps, so may as well leave them in. Also boost has a lot of negative/fail tests, not quite sure how to implement these. I was going to just add a parameter to the boost.rules.tools extension that's used for testing and leave it to not test the ones that are meant to fail until I or someone else knows a good way to fail-test and verify the expected compile/run failure did occur!

Reply:
The question about extra deps gets solved by the umbrella / cyclic dep structure recently implemented, the deps will only be downloaded and built by the test targets, and only the overarching boost dep which aliases to all the modules is needed. Yay!